### ISSUE TITLE
pagination positioning

### ISSUE THREAD

**Issue by frissdiegurke** [NodeBB/nodebb-theme-persona#68]
hey, if you use pagination instead of infinite scrolling the pagination is on the very bottom.

I think it should belong above the `.bottom-post-bar` since this is page-independent.

The way it is I don't tend to notice the pagination at all sometimes.

**Comment by barisusakli** [comment]
Yeah I think we need another pagination block near the top of the page, not sure what the best place is yet though.
